README.md

```diff
@@ -1,20 +1,33 @@
+![Maven Central](https://img.shields.io/maven-central/v/io.github.almighty-satan.slams/slams-parser-jackson?style=flat-square)
+![GitHub](https://img.shields.io/github/license/Almighty-Satan/SLAMS?style=flat-square)
+![CI](https://img.shields.io/github/actions/workflow/status/Almighty-Satan/SLAMS/gradle-build.yml?branch=master&style=flat-square)
+![Last Commit](https://img.shields.io/github/last-commit/Almighty-Satan/SLAMS?style=flat-square)
+
 # SLAMS - Simple Language And Message System
 
+___
+
+SLAMS is an easy to use message/language system allowing projects to easily support multiple languages with placeholders and message formatting. Messages can be automatically loaded from formats like JSON or supplied by the user via a custom implementation of the `LanguageParser` interface.
+
+SLAMS has been created with support for Minecraft servers in mind and therefore also supports formats like [MiniMessages](https://docs.advntr.dev/minimessage/index.html).
+
 ### Standalone Example
 // TODO
 
 ### Mini-Message Example
 ```java
-LanguageManager langManager = LanguageManager.create("English"); // Set English as the default language
-langManager.load("English", JacksonParser.createJsonParser("messages.json")); // Register language and load messages from JSON file
+Slams slams = Slams.create("English"); // Set English as the default language
+slams.load("English", JacksonParser.createJsonParser("messages.json")); // Register language and load messages from JSON file
 
-MMStringEntry test0 = MMStringEntry.of("test0", langManager, null); // Just a simple message
-MMStringEntry test1 = MMStringEntry.of("test1", langManager, ctx -> Placeholder.unparsed("hello", "world")); // Message with placeholder, "hello" will be replaced with "world"
+AdventureMessage test0 = AdventureMessage.of("test0", slams, null); // Just a simple message
+AdventureMessage test1 = AdventureMessage.of("test1", slams, ctx -> Placeholder.unparsed("hello", "world")); // Message with placeholder, "hello" will be replaced with "world"
```

Comment:
```suggestion
AdventureMessage test0 = AdventureMessage.of("test0", slams); // Just a simple message
AdventureMessage test1 = AdventureMessage.of("test1", slams, ctx -> net.kyori.adventure.text.minimessage.tag.resolver.Placeholder.unparsed("hello", "world")); // Message with placeholder, "hello" will be replaced with "world"

slams.load("English", JacksonParser.createJsonParser("messages.json")); // Register language and load messages from JSON file
```
Messages have to be registered before loading the language

Reply:
Fixed in https://github.com/Almighty-Satan/SLAMS/pull/1/commits/5cf21cccc544f367c867c1d3be642d370f90c804